.gitignore

```diff
@@ -127,3 +127,9 @@ tests/end_to_end_integration/submit_e2e_job_k8s.yml
 
 # redundant lock files that should be regenerated before builds
 docker/*/requirements.txt
+
+# wandb
+**/wandb
```

Comment:
i worry this will hide code directories named `wandb`. can you move these additions to `projects/microphysics/.gitignore`?